### DISCUSSION TITLE
matchzy_demo_upload_url

### DISCUSSION THREAD

**Discussion by sivert-io** [PhlexPlexico/G5API#289] [Help]
Hey, Phlex! Hope you are doing well :)
I'm very happy with your work and was wondering if I need to define this parameter `matchzy_demo_upload_url` in Matchzy to enable demo uploads? Does Get5Web/G5API support demo downloads?

Cheers,
Sivert

**Comment by PhlexPlexico** [comment]
Hey!

If MatchZy has backported how get5 used to handle demo uploads, then yes. G5API does support demo uploads on the `/v2/demo` as a `POST` call. I haven't really kept up with development of MatchZy as I've been occupied with other projects, but I believe the developers there have tried to keep backwards compatibility with get5 consistent.

I think if you provide `https://yourhost.xyz/api/v2/demo` as your `matchzy_demo_upload_url`, it _should_ allow uploads, provided the config value `uploadDemos` is set to true.

Hope this answers your question!

Thanks!

**Reply by sivert-io** [reply]
Thanks for the quick response! I'll give it a try and comment an update :D

**Reply by PhlexPlexico** [reply]
Just did a quick look in their codebase, and it appears that it should be supported! Just make sure you have the API key set - either `matchzy_demo_upload_header_key` or `get5_demo_upload_header_key` - this is the API key for the match. It looks like it does it the exact same way that get5 has done it, so it should work :)
